docs/index.rst

```diff
@@ -0,0 +1,15 @@
+.. documentation documentation master file, created by
```

Comment:
This file needs to be in `rst` while we have the others in `Markdown`?

Reply:
Yes. We can't define a table of content in markdown because readthedocs and sphinx do not understand it. So for every file where I need a table of content, I will be defining them as `rst`.